content/applications/general/auth/azure.rst

```diff
@@ -2,9 +2,172 @@
+later.
+
+After finishing this step, click on :guilabel:`Endpoints` on the top menu and click the *copy icon*
+next to :guilabel:`OAuth 2.0 authorization endpoint (v2)` field. Paste this value in the clipboard /
+notepad.
+
+The value should equal `https://login.microsoftonline.com/<directory_id>/oauth2/v2.0/authorize`.
+Replace the `<directory_id>` with the :guilabel:`Directory (tenant) ID` under the
+:guilabel:`Essentials` section of the *Overview* page if it is not already present in the :abbr:`URL
+(uniform resource locator)`.
+
+.. example::
+   Should the :guilabel:`Directory (tenant) ID` be equal to `6729e9df-afbb-4522-a876-f1408d416396`
+   then the new value of the :guilabel:`OAuth 2.0 authorization endpoint (v2)` :abbr:`URL (Uniform
+   Resource Locator)` should be:
+   `https://login.microsoftonline.com/6729e9df-afbb-4522-a876-f1408d416396/oauth2/v2.0/authorize`.
+
+.. image:: azure/overview-azure-app.png
+   :align: center
+   :alt: Application ID and OAuth 2.0 authorization endpoint (v2) credentials.
+
+Odoo setup
+----------
+
+Finally the last step in the Microsoft Azure OAuth configuration is to configure some settings in
+Odoo. Navigate to :menuselection:`Settings --> Integrations --> OAuth Authentication` and check the
+box to activate the OAuth login feature. Click :guilabel:`Save` to ensure the progress is saved.
+Then, sign in to the database once the login screen loads.
+
+Once again, navigate to :menuselection:`Settings --> Integrations --> OAuth Authentication` and
+click on :guilabel:`OAuth Providers`. Now, select :guilabel:`New` in the upper left corner and name
+the provider `Azure`.
+
+Paste the :guilabel:`Application (client) ID` from the previous section into the :guilabel:`Client
+ID` field. After completing this, paste the new :guilabel:`OAuth 2.0 authorization endpoint (v2)`
+value into the :guilabel:`authorization URL` field.
+
+For the :guilabel:`UserInfo URL` field, paste the following :abbr:`URL (Uniform Resource Locator)`:
+`https://graph.microsoft.com/oidc/userinfo`
+
+In the :guilabel:`Scope` field, paste the following value: `openid profile email`. Next, the Windows
+logo can be used as the CSS class on the login screen by entering the following value: `fa fa-fw
+fa-windows`, in the :guilabel:`CSS class` field.
+
+Check the box next to the :guilabel:`Allowed` field to enable the OAuth provider. Finally, add
+`Microsoft Azure Login` to the :guilabel:`Login button label` field. This text will appear next to
+the Windows logo on the login page.
+
+.. image:: azure/odoo-provider-settings.png
+   :align: center
+   :alt: Odoo provider setup in the Settings application.
+
+:guilabel:`Save` the changes should this step be required by the version of Odoo.
+
+User experience flows
+---------------------
+
+To sign in to Odoo for the first time using the Microsoft Azure OAuth provider, navigate to the Odoo
+database login page (using the invitation link) and click on the option labeled :guilabel:`Microsoft
```

Comment:
can't we just say <odoo-database-url>/web?
```suggestion
database login page (using the invitation link) and click on the option labeled :guilabel:`Microsoft
```

Reply:
This is a specific flow that the user needs to follow. They should be in the "password set" phase for the Odoo user. I don't think saying web here will be helpful, but I'm open to ideas. Can you specify where 'web' should be inserted?